src/main/java/org/kiwiproject/champagne/resource/TaskResource.java

```diff
@@ -0,0 +1,263 @@
+package org.kiwiproject.champagne.resource;
+
+import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.Collectors.toMap;
+import static java.util.stream.Collectors.toSet;
+import static org.kiwiproject.jaxrs.KiwiStandardResponses.standardNotFoundResponse;
+import static org.kiwiproject.search.KiwiSearching.zeroBasedOffset;
+
+import java.util.List;
+import java.util.Set;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.kiwiproject.champagne.core.manualdeployment.DeploymentTaskStatus;
+import org.kiwiproject.champagne.core.manualdeployment.Release;
+import org.kiwiproject.champagne.core.manualdeployment.ReleaseStatus;
+import org.kiwiproject.champagne.core.manualdeployment.Task;
+import org.kiwiproject.champagne.core.manualdeployment.TaskStatus;
+import org.kiwiproject.champagne.jdbi.DeploymentEnvironmentDao;
+import org.kiwiproject.champagne.jdbi.ReleaseDao;
+import org.kiwiproject.champagne.jdbi.ReleaseStatusDao;
+import org.kiwiproject.champagne.jdbi.TaskDao;
+import org.kiwiproject.champagne.jdbi.TaskStatusDao;
+import org.kiwiproject.jaxrs.exception.JaxrsNotFoundException;
+import org.kiwiproject.spring.data.KiwiPage;
+
+import com.codahale.metrics.annotation.ExceptionMetered;
+import com.codahale.metrics.annotation.Timed;
+import com.google.common.annotations.VisibleForTesting;
+
+import lombok.AllArgsConstructor;
+
+@Path("/manual/deployment/tasks")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+@AllArgsConstructor
+public class TaskResource {
+
+    private final ReleaseDao releaseDao;
+    private final ReleaseStatusDao releaseStatusDao;
+    private final TaskDao taskDao;
+    private final TaskStatusDao taskStatusDao;
+    private final DeploymentEnvironmentDao deploymentEnvironmentDao;
+
+    @Path("/releases")
+    @ExceptionMetered
+    @Timed
+    @GET
```

Comment:
I think we ought to have a "standard" order or grouping, so that the "most important" things are first, i.e.:

```java
@GET
@Path("/releases")
@ExceptionMetered
@Timed
```

This way the first thing you see is the HTTP method and path.